manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.installer.yaml

```diff
@@ -0,0 +1,35 @@
+# Created with WinGet Releaser v2 using Komac v1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ONLYOFFICE.DesktopEditors
+PackageVersion: 7.3.3.99
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+UpgradeBehavior: install
+FileExtensions:
+- csv
+- djvu
+- doc
+- docx
+- epub
+- odp
+- ods
+- odt
+- pdf
+- ppt
+- pptx
+- rtf
+- txt
+- xls
+- xlsx
+- xps
+ReleaseDate: 2023-03-17
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v7.3.3.99/DesktopEditors_x86_test.exe
+  InstallerSha256: F2CA1BB6C7E907D06DAFE4687E579FCE76B37E4E93B7605022DA52E6CCC26FD2
+- Architecture: x64
+  InstallerUrl: https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v7.3.3.99/DesktopEditors_x64_test.exe

```

Comment:
Actual link is "https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v7.3.3/DesktopEditors_x86.exe"